mapit_gb/management/commands/mapit_UK_import_soa_2011.py

```diff
@@ -0,0 +1,102 @@
+# This script is used to import boundary polygons and other information
+# from shapefiles representing areas of census geography
+
+# Run as: ./manage.py mapit_UK_import_soa_2011 shapefile.shp --type=lsoa
+# --commit
+
+from django.core.management.base import LabelCommand
+from django.contrib.gis.gdal import DataSource
+from mapit.models import Area, Generation, Country, Type, NameType, CodeType
+from django.conf import settings
+
+
+class Command(LabelCommand):
+    help = 'Creates Super Output Area boundaries from ONS/Scot Gov/OSNI shapefiles'
+    label = '<ONS SOA shapefile>'
+
+    def add_arguments(self, parser):
+        super(Command, self).add_arguments(parser)
+        parser.add_argument('--commit', action='store_true',
+                            dest='commit', help='Actually update the database')
+        parser.add_argument('--type', dest='file_type',
+                            help="options: lsoa, msoa, soa, dz, lz")
```

Comment:
I think there's a way to have an enum input argument?

Reply:
Added!